conf_files/huntsman.yaml

```diff
@@ -51,6 +51,12 @@ mount:
     driver: bisque
     model: 45
     template_dir: resources/bisque
+filterwheel:
+  priority:
+    one: 1
+    deux: 2
+    drei: 3
+    quattro: 4
```

Comment:
We specify which filters are on each filterwheel in `device_info.yaml`, but I don't think its appropriate to put the priorities in there as there would be a lot of duplication.

Reply:
@AnthonyHorton  I used a dictionary to make it easier if different cameras have different filters on them. Thats also the same reason I use `priority`, because if a particular filter is missing then the next-highest priority filter will be used instead. The way I have currently implemented it means that it can be more time efficient in the case of mixed filters because cameras without a particular filter aren't idle while waiting for other cameras to finish their flat fields - that's why I create a separate `DitheredObservation` for each camera. Using `flat_order` may make sense though.